rtsp.go

```diff
@@ -145,16 +145,16 @@ func (conf *Config) parseAndFixAddress(ctx context.Context, logger logging.Logge
 	}
 
 	if u.Hostname() == "UPNP_DISCOVER" {
-		host, err := viamupnp.FindHost(ctx, logger, conf.Query)
+		hosts, _, err := viamupnp.FindHost(ctx, logger, []viamupnp.DeviceQuery{conf.Query}, false)
 		if err != nil {
 			return nil, err
 		}
 
 		p := u.Port()
 		if p == "" {
-			u.Host = host
+			u.Host = hosts[0]
```

Comment:
Are we ok to hardcode that the first host in the list returned from `viamupnp.FindHost` is the one we want? Or maybe its not that big of deal because we will eventually get rid of the `UPNP_DISCOVER` special case host?

Reply:
yeah ideally we get rid of UPNP_DISCOVER down the line. though there might be some world that we dont and instead lean into using UPNP_DISCOVER, but thats a future question